manuals/1.0/en/reference.md

```
@@ -1,229 +1,194 @@
+| tag | optional | string | Classification tag | `"blog post"` |
+| doc | optional | object/string | Detailed explanation | `"Detailed description"` |
 
-There are four types in total.
+*1: Either id or href is required
 
-> What is idempotent?
->
-> It means that the result of an operation is the same whether it is performed once or multiple times. For example, adding a resource is not idempotent, but changing or deleting a resource is idempotent.
+Details of each attribute:
 
+* **id**: Unique identifier (mutually exclusive with href)
```

Markdownlint

Expected: dash; Actual: asterisk
Unordered list style

(MD004, ul-style)